ios/Classes/FacebookModule.m

```diff
@@ -359,6 +391,10 @@ -(void)authorize:(id)args
     BOOL allowUI = args == nil ? YES : NO;
     NSArray *permissions_ = permissions == nil ? [NSArray array] : permissions;
     FBSDKLoginManager *loginManager = [[[FBSDKLoginManager alloc] init] autorelease];
+    [loginManager setLoginBehavior:FBSDKLoginBehaviorBrowser];
```

Comment:
If `FBSDKLoginBehaviorBrowser` is already the default behavior, we don't need to set it here before, am I correct?